### PR TITLE
Node 10 and Upstream Updates

### PR DESCRIPTION
**Pull request by denniswebb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "start": "./node_modules/.bin/ts-node ./src/index.ts",
     "dev": "./node_modules/.bin/nodemon --exec ./node_modules/.bin/ts-node --type-check --no-cache -- ./src/index.ts",
-    "test": "yarn test-ts && ./node_modules/.bin/tsc --noEmit && yarn lint",
     "license-checker": "! license-checker --csv | grep -v -f .licenseignore | grep GPL && echo No GPL licenses found.",
     "snyk": "snyk test --org=UpsideTravel",
+    "test": "yarn test-ts && ./node_modules/.bin/tsc --noEmit && yarn lint",
```
Comment (clrung) [inline package.json]: where is this used?

File: src/repliers/dashboard_query_runner.ts [hunk 2]
```diff
@@ -51,6 +51,7 @@ export class DashboardQueryRunner extends QueryRunner {
                    element.result_maker.filterables[0].listen) {
           element.result_maker!.filterables![0].listen.forEach((listen) => {
             if (this.filters[listen.dashboard_filter_name]) {
+              if (!queryDef.filters) { queryDef.filters = {} }
```
Comment (jaygorrell) [inline src/repliers/dashboard_query_runner.ts]: looks like they've been hard at work upstream

File: package.json (hunk 1 above)
Comment (denniswebb) [reply package.json]: Upstream

File: src/repliers/dashboard_query_runner.ts (hunk 2 above)
Comment (jaygorrell) [reply src/repliers/dashboard_query_runner.ts]: actually there's a lot more in the commit history... maybe those were already brought in but not with the right sha?

File: package.json (hunk 1 above)
Comment (philarmour) [reply package.json]: +:joy:

**Pull request merged**